lib/default.nix

```diff
@@ -168,4 +168,30 @@ in {
         gomod2nix --outdir "$CURDIR"
       '';
     };
+  hermesModuleConfigToml = {modules}:
```

Comment:
Can you leave a comment about how this is intended to be used? Specifically i think it evaluates to an attrset that contains the config.toml, so you might want to point out which attr key to use to access the config.toml

Reply:
yes, will do.